src/katsdpnetboxutilities/system/config/sarao/netbox

```diff
@@ -0,0 +1,3 @@
+token=00832b01cd30c211d3e098a97f015f2a81b8f51d
```

Comment:
This is not good you have posted your token here. You should have this file as an example only. Or create a netbox.example file and put your config in ~/.config/sarao. Then use docker volume to access that.